### PR TITLE
Add March 28 news article about new Julia lectures and style update

### PR DESCRIPTION
**Pull request by natashawatkins**

### CODE REVIEW

**Comment by mmcky** [comment]
@natashawatkins this is looking great. 
It's great that you ran ``make news`` to generate the output files. 
Happy for it to be one news item. But when we tweet it might be best to keep it simpler and have a tweet per new lecture. What do you think?

**Comment by natashawatkins** [comment]
I actually already tweeted it. @jstac said he'd ideally like a single post. In the tweet, I've just named the general topics and linked to the julia table of contents.

**Comment by mmcky** [comment]
OK -- cool.

File: lecture_site/news.rst [hunk 1]
```diff
@@ -4,10 +4,26 @@
 News & Announcements
 *************************
 
+28-March-2017
+-------------
+
+You can now find three new lectures on the Julia side of QuantEcon lectures:
```
Comment (natashawatkins) [inline lecture_site/news.rst]: Really? How do I view the rendered version again? I could only find news_snippet.html

Comment (mmcky) [inline lecture_site/news.rst]: you could check with ``pandoc -s news.rst -o news.html`` it just won't show the theme.

Comment (mmcky) [inline lecture_site/news.rst]: ``news_snippet.html`` is used on the frontpage.

**Comment by natashawatkins** [comment]
Ah right I see what's happened

**Comment by mmcky** [comment]
@natashawatkins happy for this to be merged?

**Comment by natashawatkins** [comment]
Yes! I just added 'the' in front of Lake model.

**Pull request merged**